### ISSUE TITLE
Add suggestion to set FLY_APP_NAME if you're not running on Fly

### ISSUE THREAD

**Issue by allyxcristiano** [supabase/supavisor#97]
# Improve documentation

## Link 

https://github.com/supabase/supavisor/wiki/Installation-and-Usage

## Describe the problem

On this page, it's assuming the deployment will be made using Fly, but during some tests locally using `MIX_ENV=prod`, we got the error `Invalid node name! Please check your configuration` when starting the application.

## Describe the improvement

Clarify that if you're not using Fly, set the env FLY_APP_NAME ([this var is automatically set if you're using Fly](https://fly.io/docs/reference/runtime-environment/#fly_app_name))env var;

## Additional context

[eenv.sh.eex](https://github.com/supabase/supavisor/blob/main/rel/env.sh.eex#L13)

Full error due the missing of FLY_APP_NAME:

```
=INFO REPORT==== 23-May-2023::18:42:21.198353 ===
Invalid node name!
Please check your configuration

=SUPERVISOR REPORT==== 23-May-2023::18:42:21.198873 ===
    supervisor: {local,net_sup}
    errorContext: start_error
    reason: {'EXIT',nodistribution}
    offender: [{pid,undefined},
               {id,net_kernel},
               {mfargs,{net_kernel,start_link,
                                   [#{clean_halt => true,name => '@127.0.0.1',
                                      name_domain => longnames,
                                      supervisor => net_sup}]}},
               {restart_type,permanent},
               {significant,false},
               {shutdown,2000},
               {child_type,worker}]

=CRASH REPORT==== 23-May-2023::18:42:21.200453 ===
  crasher:
    initial call: net_kernel:init/1
    pid: <0.2015.0>
    registered_name: []
    exception exit: {error,badarg}
      in function  gen_server:init_it/6 (gen_server.erl, line 835)
    ancestors: [net_sup,kernel_sup,<0.2001.0>]
    message_queue_len: 0
    messages: []
    links: [<0.2012.0>]
    dictionary: [{longnames,true}]
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 28
    reductions: 327
  neighbours:

=SUPERVISOR REPORT==== 23-May-2023::18:42:21.200885 ===
    supervisor: {local,kernel_sup}
    errorContext: start_error
    reason: {shutdown,
                {failed_to_start_child,net_kernel,{'EXIT',nodistribution}}}
    offender: [{pid,undefined},
               {id,net_sup},
               {mfargs,{erl_distribution,start_link,[]}},
               {restart_type,permanent},
               {significant,false},
               {shutdown,infinity},
               {child_type,supervisor}]

=CRASH REPORT==== 23-May-2023::18:42:21.205684 ===
  crasher:
    initial call: application_master:init/4
    pid: <0.2000.0>
    registered_name: []
    exception exit: {{shutdown,
                         {failed_to_start_child,net_sup,
                             {shutdown,
                                 {failed_to_start_child,net_kernel,
                                     {'EXIT',nodistribution}}}}},
                     {kernel,start,[normal,[]]}}
      in function  application_master:init/4 (application_master.erl, line 142)
    ancestors: [<0.1999.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.2001.0>,normal}]
    links: [<0.1999.0>,<0.1998.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 28
    reductions: 167
  neighbours:

=INFO REPORT==== 23-May-2023::18:42:21.207911 ===
    application: kernel
    exited: {{shutdown,
                 {failed_to_start_child,net_sup,
                     {shutdown,
                         {failed_to_start_child,net_kernel,
                             {'EXIT',nodistribution}}}}},
             {kernel,start,[normal,[]]}}
    type: permanent

{"Kernel pid terminated",application_controller,"{application_start_failure,kernel,{{shutdown,{failed_to_start_child,net_sup,{shutdown,{failed_to_start_child,net_kernel,{'EXIT',nodistribution}}}}},{kernel,start,[normal,[]]}}}"}
Kernel pid terminated (application_controller) ({application_start_failure,kernel,{{shutdown,{failed_to_start_child,net_sup,{shutdown,{failed_to_start_child,net_kernel,{'EXIT',nodistribution}}}}},{kernel,start,[normal,[]]}}})

Crash dump is being written to: erl_crash.dump...done
```

**Comment by J0** [comment]
Think this might be addressed by #95: https://github.com/supabase/supavisor/blob/main/rel/env.sh.eex#LL12C1-L12C1

**Comment by allyxcristiano** [comment]
TY @J0, I'll map here to remove this env var once we start using the release with this code, closing this issue
